packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -440,14 +434,34 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
     items,
   });
 
-  const isEditable = useCallback(
-    (item: T) => {
-      // If the So is `managed` it is never editable.
-      if (item.managed) return false;
-      return itemIsEditable?.(item) ?? true;
-    },
-    [itemIsEditable]
-  );
+  const tableItemsRowActions = useMemo(() => {
+    return items.reduce<TableItemsRowActions>((acc, item) => {
+      const ret = {
+        ...acc,
+        [item.id]: rowItemActions ? rowItemActions(item) : undefined,
+      };
+
+      if (item.managed) {
+        ret[item.id] = {
+          ...ret[item.id],
+          delete: {
+            enabled: false,
+            reason: i18n.translate('contentManagement.tableList.managedItemNoDelete', {
+              defaultMessage: 'This item is managed by Elastic. It cannot be deleted.',
+            }),
+          },
+          edit: {
```

Comment:
itemIsEditable was added to disable editing of legacy visualizations in serverless. Edit could be disabled for 2 reasons - its managed, or its a legacy vis and its on serverless.

Reply:
<img width="1383" alt="Screenshot 2024-02-08 at 10 12 45 AM" src="https://github.com/elastic/kibana/assets/315764/896b885a-5061-4bcd-9747-1a7cfa50e02f">

Yeah—that's preserved. Though the message is a little misleading. I don't think that's new.

Reply:
When readOnly is true for legacy visualizations, you are correct that the action to edit the title and description is blocked, the visualization name itself can still be clicked to open the editor. In the image below the chart title is a link to open an editor. This should not be allowed when `readOnly` is true.
<img width="500" alt="Screenshot 2024-02-08 at 2 47 29 PM" src="https://github.com/elastic/kibana/assets/373691/8f5ab0fa-4504-488a-be51-3788f06ac15b">

Reply:
It looks like this is no different from how it works on `main`.

What I did to repro: 
- add `vis_type_timeseries.readOnly: true` to `kibana.yml`
- run `yarn start --serverless es`

It allows you to click the link, but then you get a toast explaining why you can't get to the editor.

### `main`

https://github.com/elastic/kibana/assets/315764/c7083700-fe61-4353-b1f6-cbb7ed57a331

### This branch

https://github.com/elastic/kibana/assets/315764/23dff455-b51c-4b28-8bfb-47d402fcbe6e